numeric.h

```diff
@@ -0,0 +1,34 @@
+#ifndef NUMBER_THEORY_NUMERIC_H
+#define NUMBER_THEORY_NUMERIC_H
+
+#include <limits>
+#include <numeric>
+#include <utility>
+
+/**
+ * Common numeric functions.
+ */
+
+namespace number_theory {
+
+using std::gcd;
+using std::lcm;
+
+/**
+ * Extended Euclidean algorithm.
+ * Given two numbers a and b, return a pair (x, y)
```

Comment:
Let's use declarative comments instead of imperative ones.

Reply:
done